src/reactviews/pages/ExecutionPlan/highlightExpensiveOperations.tsx

```diff
@@ -0,0 +1,176 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useState } from 'react';
+import { ExecutionPlanContext } from "./executionPlanStateProvider";
+import * as utils from './queryPlanSetup';
+import './executionPlan.css';
+import { Button, Combobox, makeStyles, Option } from '@fluentui/react-components';
+import { Checkmark20Regular, Dismiss20Regular } from '@fluentui/react-icons';
+import * as ep from './executionPlanInterfaces';
+
+const useStyles = makeStyles({
+	inputContainer: {
+		position: "absolute",
+		top: 0,
+		right: "35px",
+		padding: "10px",
+		border: "1px solid #ccc",
+		zIndex: "1",
+		boxShadow: "0px 4px 6px rgba(0, 0, 0, 0.1)",
+		display: "flex",
+		alignItems: "center",
+		gap: "2px",
+		opacity: 1
+	},
+	dropdown: {
+		maxHeight: "200px"
+	}
+})
+
+interface HighlightExpensiveOperationsProps {
+	executionPlanView: any;

```

Comment:
Can we use proper types instead of any?

Reply:
You should be able to import and use vscodeMssql types here.

Reply:
added proper typings!